src/routes/at-sce.routes.js

```diff
@@ -0,0 +1,28 @@
+const express = require("express");
+const router = express.Router();
+const path = require("path");
+
+// SCE Controller
+const {
+  renderSigninForm,
+  signin,
+  signout,
+  calculator,
+} = require("../controllers/at-sce.controller");
+
+// ============= Sub Routes =============
+
+// AT-SCE - Index
+router.get("/", renderSigninForm);
+
+// AT-SCE - Login
+router.get("/signin", renderSigninForm);
+router.post("/signin", signin);
+
+// AT-SCE - Logout
+router.get("/signout", signout);
```

Comment:
signout should be a post

Reply:
I think the "GET" path is fine here (it does not refer to the HTTP operation)